randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -2286,6 +2286,8 @@ Extra - asset_id: collision_camera_034
           Any of the following:
               Gravity Suit or Lay Cross Bomb
```

Comment:
Unrelated, but I noticed this still has Cross Bombs for trickless. Could probably open another pr addressing that, or put it in this one.